app/gui/src/project-view/components/DocumentationEditor.vue

```diff
@@ -80,13 +82,21 @@ const editorMarkdown = computed(() =>
 const editorContent = computed(() => unwrapOr(editorMarkdown.value, undefined))
 
 const { syncExt, connectSync } = useYTextSync(editorContent)
+const editorPersistenceExt = editorPersistence({
+  documentViewId: useDocumentViewId({
+    projectId,
+    methodPointer: () => unwrapOr(currentProject.ref.value?.graph.currentMethod.pointer, undefined),
+    view: 'DocumentationEditor',
+  }),
+  scroll: { y: true, x: false },
+})
 </script>
 
 <template>
   <div class="DocumentationEditor">
     <MarkdownEditor
       v-if="currentMethodAst.ok"
-      :extensions="syncExt"
+      :extensions="[syncExt, editorPersistenceExt]"
```

Comment:
Let's make this array into a computed value, so we don't get a needless extension list update every render.